lib/googmodule.js

```diff
@@ -24,14 +24,15 @@ var createPreprocesor = function(logger, /* config.basePath */ basePath) {
 
     log.debug('Processing "%s".', file.originalPath);
 
-    // Path must include http:// protocol and '/base/' so that Karma can still recognize the URL
-    // and apply source maps.
-    content =
-        JSON.stringify(content + '\n//# sourceURL=http://googmodule/base/' + relativePath + '\n');
-    // Make sure not to insert any line breaks before the content, so that line
-    // numbers match the original. It's a poor man's source map, essentially.
-    var output = '/* Generated from ' + relativePath + ' by karma-googmodule-preprocessor */ ' +
-                 'goog.loadModule(' + content + ');\n';
+    var output = `/* Generated from ${relativePath} by karma-googmodule-preprocessor */
+goog.loadModule(function(exports) {
+"use strict";
+${content}
+//# sourceURL=http://googmodule/base/${relativePath}
```

Comment:
This sourceURL comment must be the last line.  The tools work backwards looking at the comments at the end.

Reply:
Per my comment above, this isn't doing anything regardless of where it is placed. I still see the wrapper when debugging the file.